dbms/src/Storages/MergeTree/MergeTreePartInfo.h

```diff
@@ -88,4 +88,11 @@ struct MergeTreePartInfo
     static constexpr UInt32 MAX_BLOCK_NUMBER = 999999999;
 };
 
+/// Information about detached partition, which includes its prefix in
```

Comment:
We have clear difference between "parts" and "partitions". (Partition is a set of parts.)